src/invoice2data/extract/templates/nl/nl.praxis.yml

```diff
@@ -16,8 +16,15 @@ fields:
     parser: regex
     regex:
       - '(\d{1,3}[-]\d{5,6})'
-  date: (\d{1,2}[-|\/]\d{2}[-|\/]\d{4})
-  invoice_number: (\d{1,2}[-|\/]\d{2}[-|\/]\d{4})
+  date:
+    parser: regex
+    regex: 
+    - \s(\d{1,2}.\d{2}.\d{4})\s
+    - \s(\d{4}-\w{3}-\d{2})\s
```

Comment:
Nitpicking: I'd suggest to increate indent of `regex` entries (add 2 extra spaces)